docs/about/participating.rst

```diff
@@ -21,20 +21,20 @@ CIViC Principles
 
 How to Contribute
 ~~~~~~~~~~~~~~~~~
-There are several ways you can make a contribution to this important problem:
+There are several ways you can make a contribution to this important cause:
 
 
-- **Add:** Add evidence statements that support clinical actions associated with genomic variants (e.g. single nucleotide substitution, structural variant, gene fusion, etc.), add summaries of the corpus of evidence for a variant, suggest publications that should be curated, or help create a synthesized interpretation (i.e., Assertion) of the evidence supporting a variant in the context of a disease and its clinical implications.
+- **View:** Make use of the community-created content in your own research by browsing, searching, and examining detailed evidence items. All CIViC data and source code are provided freely for almost any use.
 
-- **Approve:** Community Editors may approve submitted content and elevate the status of this content or approve suggest changes (edits), improving the existing content.
+- **Discuss:** Participate in an ongoing discussion in an effort to reach community consensus on the appropriate clinical action(s) for a genomic event via comments.
 
-- **View:** Make use of the community-created content in your own research by browsing, searching, and examining detailed evidence items. All CIViC data and source code are provided freely for almost any use.
+- **Flag:** Call Editor attention to Entities (Variants, Genes, Evidence, Assertions) that contain inaccuracies or inconsistencies.
 
-- **Edit:** Submit a correction or addition to any details about a genomic event, evidence statement, or interpretation.
+- **Suggest Revisions:** Submit a correction or addition to any details about a Gene, Variant, Evidence Etatement, or Assertion.
```

Comment:
I do wonder if we should say "(Edit)" to further emphasize this point. This is why we didn't use CIViC jargon in the initial version of this.